src/map/npc.c

```diff
@@ -4795,3 +4802,28 @@ void do_init_npc(void){
 	map_addiddb(&fake_nd->bl);
 	// End of initialization
 }
+
+/**
+* Check if the player can sell the current item
+* @param sd map_session_data of the player
+* @param item struct of the checking item.
+* @return bool 'true' is sellable, 'false' otherwise
+*/
+bool npc_can_sell_item(struct map_session_data * sd, struct item * item) {
```

Comment:
Just from the description this sounds more like a pc_ function than a npc_ function.